data/mods/Aftershock/items/afs_harvest.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "zombie",
+    "//": "your bog standard zombie",
+    "type": "harvest",
+    "entries": [ { "drop": "afs_titanium_implant", "type": "bionic", "flags": [ "FILTHY", "NO_STERILE" ], "max": 1 } ]
```

Comment:
Out of curiosity how often do these items drop when butchering?  Or does the body have to be dissected?

Reply:
So these come out while dissected due to the bionic type. I wasn't sure how the weighting and probability etcetera was for normal butchering, but knew that CBM's were quite infrequent and picky so I used the pheromone json as a precedent.

Reply:
OK so you'd need to be looking for titanium to dissect zombies in hopes of finding it. I like it.